.gitignore

```diff
@@ -8,3 +8,4 @@
 /vendor/
 /bin/
 /composer.phar
+/nbproject/private/
```

Comment:
I think this is a very personal and we should avoid it. BTW, many of devs use PhpStorm instead